app/main.py

```diff
@@ -34,21 +34,25 @@ def __truediv__(self, other: int) -> Distance:
         return Distance(round(self.km / other, 2))
 
     def __lt__(self, other: Union[Distance, int, float]) -> bool:
-        other = self.validation(other)
+        other = Distance.validation(other)
         return self.km < other
 
     def __gt__(self, other: Union[Distance, int, float]) -> bool:
-        other = self.validation(other)
-        return self.km > other
+        other = Distance.validation(other)
+        return not self.__le__(other)
 
     def __eq__(self, other: Union[Distance, int, float]) -> bool:
-        other = self.validation(other)
+        other = Distance.validation(other)
         return self.km == other
 
     def __le__(self, other: Union[Distance, int, float]) -> bool:
-        other = self.validation(other)
+        other = Distance.validation(other)
```

Comment:
In this code i use distance.validation(other) instead of self.validation(other) because validation is a static method, static methods belong to the class, not to its instances, I did this to emphasize that the method is static and can be called through the class